web/src/gql/queries/item.ts

```diff
@@ -121,43 +121,45 @@ export const GET_ITEMS_BY_IDS = gql`
 `;
 
 export const SEARCH_ITEM = gql`
-  query SearchItem(
-    $query: ItemQueryInput!
-    $sort: ItemSortInput
-    $filter: ConditionInput
-    $pagination: Pagination
-  ) {
-    searchItem(input: { query: $query, sort: $sort, filter: $filter, pagination: $pagination }) {
+  query SearchItem($searchItemInput: SearchItemInput!) {
+    searchItem(input: $searchItemInput) {
       nodes {
         id
         title
         schemaId
         createdAt
         updatedAt
         status
+        version
         assets {
           id
           url
         }
-        createdBy {
-          ... on Integration {
-            name
-          }
-          ... on User {
-            name
-          }
-        }
```

Comment:
Why did you remove createdBy?

Reply:
When graphQL API that requests createdBy user quits the local server(http://localhost:8080/) on my local environment. 
It will add createBy and updatedBy when this is fixed.